testsuite/features/step_definitions/retail_steps.rb

```diff
@@ -442,6 +476,15 @@ def compute_image_name
 When(/^I enter the MAC address of "([^"]*)" in (.*) field$/) do |host, field|
   if host == 'pxeboot_minion'
     mac = $pxeboot_mac
+  elsif host == 'sle11sp3_terminal'
+    mac = $sle11sp3_terminal_mac
+    mac = 'EE:EE:EE:00:00:05' if mac.nil?
```

Comment:
```suggestion
    mac = "EE:EE:EE:00:00:0${ADDRESSES[host]}" if mac.nil?
```

Reply:
I can't see where we define variables of that kind $sle11sp3_terminal_mac, but maybe we can reduce the cyclomatic complexity of that step if we can something like:
```
elsif host.include? 'terminal'
  mac = TERMINAL_MAC.has_key?(host) ? TERMINAL_MAC[host] : "EE:EE:EE:00:00:0${ADDRESSES[host]}"
```

Reply:
less readable :-(

Reply:
For BV, whatever helps us introducing more client types with less effort and less maintenance is welcomed. When adding something new to BV, we need to always think if we can make it in a way that if tomorrow we have a new client, we don't need to do too many changes here and there. And even like that... we will need :D

Reply:
It's definitely more readable this way.